kubeadm/flannel/flannel-host-gw.yml

```diff
@@ -74,6 +93,50 @@ data:
         ]
       }
     }
+  cni-conf-containerd.json: |
+    {
+      "cniVersion": "0.2.0",
+      "name": "cbr0",
+      "type": "flannel",
```

Comment:
@nagiesek
win-bridge and win-overlay support "loopbackDSR" 
https://github.com/containernetworking/plugins/tree/master/plugins/main/windows/win-bridge

- is this also implemented in the SDN plugins and should be added if DSR is enabled?

Reply:
This implementation relies on wins and I'm not sure if there is a great way to check if DSR is enabled.
Let me make an another issue in this repo for DSR enablement and we can merge that support with a seperate PR.